tests/js/spec/views/stream.spec.jsx

```diff
@@ -395,7 +395,7 @@ describe('Stream', function() {
         context,
       }).instance();
 
-      let actual = stream.getInitialState();
+      let actual = stream.state;
       expect(_.pick(actual, _.keys(expected))).toEqual(expected);
```

Comment:
I know this isn't your code, just general fyi, we could use `expect.objectContaining()` here.

Reply:
Will update to this soon